Sources/Crypto/RSAUtils.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import CommonCrypto
+
+public class RSAUtils {
```

Comment:
Do you mind fixing the indentation of this class?

Reply:
sure, done